seatunnel-connectors-v2/connector-http/connector-http-base/src/main/java/org/apache/seatunnel/connectors/seatunnel/http/config/HttpParameter.java

```diff
@@ -25,18 +25,25 @@
 import java.util.Map;
 import java.util.stream.Collectors;
 
+import static org.apache.seatunnel.connectors.seatunnel.http.config.HttpConfig.PAGE_NO;
+import static org.apache.seatunnel.connectors.seatunnel.http.config.HttpConfig.PAGE_NO_FIELD;
+import static org.apache.seatunnel.connectors.seatunnel.http.config.HttpConfig.PAGE_SIZE;
+import static org.apache.seatunnel.connectors.seatunnel.http.config.HttpConfig.PAGE_SIZE_FIELD;
+
 @Data
```

Comment:
Use `@Getter` and `@Setter` is better, because if use this object as a key in Map `@Data` may cause some problem.